libraries/ts-command-line/src/CommandLineParameterProvider.ts

```diff
@@ -81,8 +81,17 @@ abstract class CommandLineParameterProvider {
   }
 
   protected defineOptionParameter(options: ICommandLineOptionDefinition): CommandLineOptionParameter {
+    if (!options.options || options.options.length < 2) {
+      throw new Error(`When defining an option parameter, the options array must`
```

Comment:
>options [](start = 62, length = 7)

This is a confusing name. A command line "option" is usually just a string parameter, or maybe it means any general parameter.  If there are a fixed set of choices, that's usually called an "enum".

If this is a disruptive change, we can handle it in a separate PR